sdks/python/apache_beam/io/external/generate_sequence.py

```diff
@@ -50,11 +50,12 @@ class GenerateSequence(ptransform.PTransform):
     Experimental; no backwards compatibility guarantees.
   """
 
+  URN = 'beam:external:java:generate_sequence:v1'
```

Comment:
Should be of the format "beam:transforms:xlang..." ?

Reply:
There are other places where ‘beam:external:java’ format is used. It would be better to address all of them in one commit later if we have to.